### ISSUE TITLE
Can use the server for hosting games on Facebook?

### ISSUE THREAD

**Issue by islandlover1980** [kzahel/web-server-chrome#48]
To whom it may concern,
I wanted to know, if I can use the program to serve on Facebook?
I was looking into using this app for hosting games.
Is there any advice or suggestions to what I can do?
Thanks,
islandlover1980

**Comment by kzahel** [comment]
I don't think so. But maybe you could let me know more details about the specific game

**Comment by islandlover1980** [comment]
Kzahel,
What I meant was, that your app is a server app right?
If it is a server app?
Could I use it as a game server for a website or on Facebook?
My OS is a Chrome OS machine.
I don't have windows software to use as a server or another computer to use as a server to host games.
If you could help me with this situation, that would be helpful?
I like the work that you've done for creating 2,000 OK! app!
Its awesome!
islandlover1980

**Comment by kzahel** [comment]
What specific game did you have in mind? I don't know much about facebook games.

Most game servers require custom software. This project only serves static files right now, so I don't see how it would work.

**Comment by islandlover1980** [comment]
Kzahel,
Could I collaborate with you on this project?
To find out way to make the app work for chromebooks?
My game that I am trying to launch on Facebook, is Turtle_mega_rush...
Your app is cool to use, and also I was wondering if I could also use it to launch a website?
Let me know you thoughts?
I will be looking forward to working with you.
Thanks,
islandlover1980

**Comment by kzahel** [comment]
my app only serves static files.
You can serve static files easily on github, even on your own domain.
Sorry I am not sure how I can help you, nor am I sure what you want to accomplish exactly.

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
